### PR TITLE
[enhance](mtmv)Mtmv support audit log

### PR DESCRIPTION
**Pull request by zddr**
mtmv not have real sql, fill stmt with a string containing taskId and partitionNames information

select * from __internal_schema.audit_log;
```
query_id: e0c4d330b49742c7-99dcf9903ad94b51
             time: 2024-09-30 15:19:05.216
        client_ip: 
             user: admin
          catalog: internal
               db: 
            state: OK
       error_code: 0
    error_message: 
       query_time: 119
       scan_bytes: 4542464
        scan_rows: 244
      return_rows: 244
          stmt_id: 0
        stmt_type: INSERT
         is_query: 0
      frontend_ip: 127.0.0.1
      cpu_time_ms: 4
         sql_hash: null
       sql_digest: 
peak_memory_bytes: 2829184
   workload_group: normal
             stmt: Asynchronous materialized view refresh task, taskId: 260885015586390, partitions refreshed by this insert overwrite: [mv_log]
```

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run performance

**Review by yiguolei — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/job/extensions/mtmv/MTMVTask.java [hunk 1]
```diff
@@ -226,20 +227,34 @@ private void exec(ConnectContext ctx, Set<String> refreshPartitionNames,
...
+                        executor.getParsedStmt(), executor.getQueryStatisticsForAuditLog(), true);
+            }
         }
     }
 
+    private String getDummyStmt(Set<String> refreshPartitionNames) {
+        return String.format(
+                "Asynchronous materialized view refresh task, "
+                        + "taskId: %s, partitions refreshed by this insert overwrite: %s",
+                super.getTaskId(), refreshPartitionNames);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/job/extensions/mtmv/MTMVTask.java:255]: Should also print table name
For example, if our partition name is p2014_01_01， we could not find the related mv task from the partition name.

**Comment by zddr** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run feut

**Review by morrySnow — APPROVED** [review]

**Pull request merged**